.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/rust
+{
+  "name": "Rust",
+  "image": "mcr.microsoft.com/devcontainers/rust:1-1-bullseye",
+  "onCreateCommand": ".devcontainer/oncreate",
+  "features": {
+    "ghcr.io/devcontainers/features/azure-cli:1": {},
+    "ghcr.io/devcontainers/features/github-cli:1": {},
+    "ghcr.io/devcontainers/features/sshd:1": {}
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "rust-lang.rust-analyzer",
+        "EditorConfig.EditorConfig",
+        "streetsidesoftware.code-spell-checker"
+      ]
```

Comment:
This should match what's in `.vscode/extensions.json` - which can certain be updated as well. I think we should also add the LLDB debugger like I do in all mine e.g., <https://github.com/heaths/template-rustlang/blob/main/.devcontainer/devcontainer.json>.

Reply:
Strange that it prompted me to add extensions here when `.vscode/extensions.json` exists. I'd kinda hope that Codespaces could just install the extensions from there... I'll remove this list for now and see if `.vscode/extensions.json` is sufficient, and I'll add the LLDB debugger and spell checker to that file.

Reply:
Well, that doesn't work. Users can obviously still install them, but there's no way that I can find to tell the codespace config to _automatically_ install extensions from `.vscode/extensions.json`. So, for now, I'll mirror the lists, I guess.

Reply:
Yeah, I brought this up with a team internally as well but never heard back. I also didn't follow up.